nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -876,9 +877,13 @@ public static Nullness getGenericMethodParameterNullness(
       int parameterIndex,
       Symbol.MethodSymbol method,
       Symbol enclosingSymbol,
+      @Nullable MemberReferenceTree memberReferenceTree,
```

Comment:
We will also need to update the Javadoc for this method to document the new parameter.

Reply:
Actually, I have a better idea.  Just below this method (line 909), there is another overload of `getGenericMethodParameterNullness()` that takes a `Type` rather than a `Symbol`.  How about we not modify this method, and instead at the call site just invoke the other method if we have a `MemberReferenceTree`?  Right now we'll have to do a null check on the `Type` at the caller, but I am also open to making the parameter of the other method `@Nullable` and doing the check there.

Reply:
if we have a MemberReferenceTree!=null, it's type will also not be null right? if I understand correctly is this what you're suggesting, the place where we call the method from NullAway.java (line 734) to be something like:

```
if (isOverriddenMethodAnnotated) {
      for (int i = 0; i < superParamSymbols.size(); i++) {
        overriddenMethodArgNullnessMap[i] =
            Nullness.paramHasNullableAnnotation(overriddenMethod, i, config)
                ? Nullness.NULLABLE
                : (config.isJSpecifyMode()
                    ? (memberReferenceTree != null)
                        ? GenericsChecks.getGenericMethodParameterNullness(
                            i,
                            overriddenMethod,
                            ASTHelpers.getType(memberReferenceTree),
                            state,
                            config)
                        : GenericsChecks.getGenericMethodParameterNullness(
                            i,
                            overriddenMethod,
                            overridingParamSymbols.get(i).owner.owner,
                            state,
                            config)
                    : Nullness.NONNULL);
      }
    }
```